client/reader/stream/index.jsx

```diff
@@ -412,6 +413,9 @@ export default class ReaderStream extends React.Component {
 	showUpdates = () => {
 		this.props.onUpdatesShown();
 		FeedStreamStoreActions.showUpdates( this.props.store.id );
+		if ( this.props.recommendationsStore ) {
```

Comment:
it happens when you click on the XX New Posts update pill that appears when new posts are available. I want to shuffle the recs because otherwise you'll see the same recs in the same order, but in a new place in the stream. Just felt weird.